inference-engine/tests/functional/shared_test_classes/src/single_layer/result.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_layer/result.hpp"
+
+namespace LayerTestsDefinitions {
+
+std::string ResultLayerTest::getTestCaseName(testing::TestParamInfo<ResultTestParamSet> obj) {
+    std::vector<size_t> inputShape;
+    InferenceEngine::Precision inputPrecision;
+    std::string targetDevice;
+    ConfigMap additionalConfig;
+    std::tie(inputShape, inputPrecision, targetDevice, additionalConfig) = obj.param;
+
+    std::ostringstream result;
+    result << "IS=" << CommonTestUtils::vec2str(inputShape) << "_";
+    result << "inPRC=" << inputPrecision.name() << "_";
+    result << "targetDevice=" << targetDevice;
+    return result.str();
+}
+
+void ResultLayerTest::SetUp() {
+    std::vector<size_t> inputShape;
+    InferenceEngine::Precision inputPrecision;
+    std::string targetDevice;
+    ConfigMap additionalConfig;
+    std::tie(inputShape, inputPrecision, targetDevice, additionalConfig) = GetParam();
+
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(inputPrecision);
```

Comment:
Am I right that you create a function:
```
Const      Parameter
     |
    \/
Result
```
Why do you need to create such network? Why do you need to have redundant operations?